.github/workflows/conformance-ginkgo.yaml

```
@@ -224,7 +264,25 @@ jobs:
             '.include |= map(select(.["k8s-version"] as $k | $prs[] | select($k == .))) + $focus[0].include |
             . + {"k8s-version": $prs} |
             .focus = $focus[0].focus | .exclude = $focus[0].exclude' \
-            main-k8s-versions.json> /tmp/merged.json
+            main-k8s-versions.json > /tmp/merged.json
+
+          # Parse flags from inputs and optionally restrict the matrix.
+          DETECT_RACE="${{ needs.setup-vars.outputs.detect-race }}"
+          ALL_TESTS=$(echo '${{ inputs.extra-args }}' | jq -r '."all-tests" // false | tostring')
+          if [ "${{ github.event_name }}" != "schedule" ] && [ "${DETECT_RACE}" = "true" ] && [ "${ALL_TESTS}" != "true" ]; then
```

Does this need to filter out schedule? I understand if we don't do race jobs with schedule, but if we don't do it then we also presumably don't need to filter it out?
```suggestion
          if [ "${DETECT_RACE}" = "true" ] && [ "${ALL_TESTS}" != "true" ]; then
```

---

Ah, you are right, I started with the != schedule until I realized we need the `ALL_TESTS` and forgot the remove the schedule.